### PR TITLE
Add Play, stop and error icons, remove unused css

### PR DESCRIPTION
**Pull request by amit-mb**
- Add icons

screen shot 2018-07-11 at 2 36 18 pm

screen shot 2018-07-11 at 2 36 30 pm

screen shot 2018-07-11 at 2 36 54 pm

Better looking font `Anonymous Pro` for code blocks;
screen shot 2018-07-11 at 3 43 48 pm

### CODE REVIEW

File: app/stylesheets/_table.css [hunk 1]
```diff
@@ -153,8 +152,8 @@
 .ReactTable .rt-tbody .rt-tr-group:last-child {
   border-bottom: 0;
 }
-.ReactTable .rt-tbody .rt-td {
-  border-right: 1px solid rgba(0, 0, 0, 0.02);
+.ReactTable .rt-tbody .rt-td:hover {
+  /* border-right: 1px solid rgba(0, 0, 0, 0.02); */
```
Comment (revathskumar) [inline app/stylesheets/_table.css]: can we remove this style?

File: app/index.html [hunk 2]
```diff
@@ -1,17 +1,21 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <meta charset="UTF-8">
   <title>Interceptor</title>
   <meta name="viewport" content="width=device-width">
   <meta name="description" content="" />
   <meta name="keywords" content="" />
   <link href="https://fonts.googleapis.com/css?family=Barlow:400,600" rel="stylesheet" />
```
Comment (revathskumar) [inline app/index.html]: DO we still need this font?

File: app/containers/Popup.tsx [hunk 3]
```diff
@@ -93,20 +96,22 @@ export class Popup extends React.Component<POPUP_PROPS & DispatchProps, {}> {
...
+    const buttonLabel = tabRecord.enabledStatus ? " Stop Listening" : " Start Listening";
+    const ButtonIcon = tabRecord.enabledStatus ? PlayIcon : StopIcon;
 
     return (
       <div className="popup">
         <header>
           <div className="grid-container">
             <Logo />
-            <button
+            <div
```
Comment (revathskumar) [inline app/containers/Popup.tsx]: Why is this converted into div? Can we keep this back to button itself?

File: app/index.html (hunk 2 above)
Comment (amit-mb) [reply app/index.html]: Yeah, the font Anonymous Pro is used only inside the textarea where json/any response is shown.. Other places, the font `Barlow` looks good

**Pull request merged**